src/Umbraco.Web.UI.Client/src/views/common/overlays/treepicker/treepicker.controller.js

```diff
@@ -20,16 +33,60 @@ angular.module("umbraco").controller("Umbraco.Overlays.TreePickerController",
             selectedSearchResults: []
         }
 
-        $scope.model.selection = [];
-
-        //Used for toggling an empty-state message
+        $scope.model.selection = [];
+
+        //Used for toggling an empty-state message
         //Some trees can have no items (dictionary & forms email templates)
         $scope.hasItems = true;
-        $scope.emptyStateMessage = dialogOptions.emptyStateMessage;
-
+        $scope.emptyStateMessage = dialogOptions.emptyStateMessage;
+
+        $scope.languages = [];
+        $scope.selectedLanguage = {};
+        $scope.page = {};
+        $scope.page.languageSelectorIsOpen = false;

```

Comment:
This also seems to be copied over from the navigation controller, please make sure that you aren't ever just copy/pasting and that you are just using that method as a reference. For consistency in this controller, the newly added `.page` variable doesn't make much sense.

Reply:
I tried to reuse the drop down from the navigation controller and, well everything that came with it made sense to stay at the time, I should have given it more thought.